examples/src/decodeData.js

```diff
@@ -5,6 +5,29 @@ import { getInstance } from './instantiation.js';
 const myERC725 = getInstance();
 
 const decodedDataOneKey = myERC725.decodeData([
+  {
+    keyName: 'LSP3Profile',
+    value:
+      '0x00006f357c6a0020820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361697066733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178',
+  },
+]);
+/**
+[
+  {
+    name: 'LSP3Profile',
+    key: '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
+    value: {
+      verification: {
+        method: 'keccak256(utf8)',
+        data: '0x820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361',
+      },
+      url: 'ipfs://QmYr1VJLwerg6pEoscdhVGugo39pa6rycEZLjtRPDfW84UAx'
+    }
+  }
+]
+*/
+
+const decodedDataOneKeyOld = myERC725.decodeData([
```

Comment:
Why do we need to keep the `decodedDataOneKeyOld`here ? Thanks

Reply:
I just removed it - no need to have examples for outdated / deprecated features.